index.bs

```diff
@@ -62,6 +66,7 @@ spec: WebCryptoAPI; urlPrefix: https://www.w3.org/TR/WebCryptoAPI/
 </pre> <!-- class=anchors -->
 
 <pre class="link-defaults">
+spec:infra; type:dfn; text:list
```

Comment:
Would y'all file an issue in https://github.com/tabatkins/bikeshed/issues about generating a dependencies section earlier in the document? For now ... I'm going to take this thread as an indication not to add "infra" to `#dependencies` unless y'all say otherwise.